shielddash/settings.py

```diff
@@ -111,6 +111,8 @@
 
 STATIC_URL = '/static/'
 
+SECURE_SSL_REDIRECT = not DEBUG
+SECURE_HSTS_SECONDS = (60 * 60 * 7 * 24 * 365)
```

Comment:
That's 7 years. Originally you had 1 year so I'd drop the 7. I think 1 year is good.